src/components/Provider/Analytics/ProviderStats/StatsFilter/StatsFilter.js

```diff
@@ -23,11 +24,20 @@ import { useIntl } from "react-intl";
  * @returns {JSX.Element} JSX Component.
  */
 const StatsFilter = ({ list, onChange }) => {
+  const baseExchanges = useExchangeList();
```

Comment:
maybe just `exchanges`?

Reply:
@cwagner22  exchanges was already defined so I named this.

Reply:
Maybe we should remove useExchangesOptions and only use useExchangeList?

Reply:
I was thinking about using userExchangeList inside useExchangeOptions.

Reply:
ok it could work too.

Reply:
@cwagner22  done.